### PR TITLE
  include tick subpages, fix tnt duping article, remove rethorical questions 

### LINKED ISSUE #13: tntDuplication has incorrect content, mentions pepole and has opinionated writing

### PR DESCRIPTION
**Pull request by pwouik**
should resolve #13, #10, #9

### CODE REVIEW

File: BugsAndExploits/TntDuping.md [hunk 1]
```diff
@@ -6,15 +7,21 @@ description: Explaination of tnt duplication
 # tnt duplication
 
 ## History
-TNT Duplication, or TNT Duping, was first discovered in 1.7 but was not used as it was slightly impractical before the addition of slime blocks. It gained popularity for the first time when upcoming technical Minecraft player ilmango released a video on YouTube entitled "The BEST Way To Make Perimeters in Minecraft," showcasing a slimestone flying machine that used flying TNT Dupers and liquid clearing machines to create a large hole fully automatically.
+TNT Duplication, or TNT Duping, was first discovered in 1.7 but was not used as it was slightly impractical before the addition of slime blocks.
```
Comment (EmiliaThorsen) [inline BugsAndExploits/TntDuping.md:10]: tnt duping this article is refering to was discoverd by myren in 1.9, it was later discoverd that you could do something similar in 1.7 and below but not 1.8

tnt duping is also extremly usefull even if you cant move the tnts with slime

File: BugsAndExploits/TntDuping.md [hunk 2]
```diff
@@ -6,15 +7,21 @@ description: Explaination of tnt duplication
...
+It gained popularity with the invention of world eaters.
+### Opposition
+Mojang historically opposes TNT Duping as they consider it an exploit, and tried to fix it, but then have decided to leave it in the game for the time being, due to a lack of replacement method
```
Comment (EmiliaThorsen) [inline BugsAndExploits/TntDuping.md:13]: they tried to fix it 4 times should be mentioned, but in 1.16 they decided to keep it until better methods was implemented

File: BugsAndExploits/TntDuping.md [hunk 3]
```diff
@@ -6,15 +7,21 @@ description: Explaination of tnt duplication
...
-These specific blocks are able to update after the list of blocks to be moved is created, but before the blocks have moved. This update can also be propagated through other blocks, as long as it is instant (for example using rails, dust, noteblocks (1.13+), or TNT itself), which allows for up to 11 TNT to be duplicated using a single piston push, the 12th block being for updating.
-In this step, the TNT entity has already been created from step 2, so now the piston actually moves the blocks it already has in the list, and this means that the TNT is preserved in block form.
-Opposition
-Mojang historically opposes TNT Duping as they consider it an exploit, but have decided to leave it in the game for the time being, 
-due to a lack of replacement method
-https://www.reddit.com/r/Minecraft/comments/fkt9jf/anchor_yourself_to_the_nether_snapshot_20w12a_is/fkv33q3?utm_source=share&utm_medium=web2x&context=3
- for what TNT duping currently does.
\ No newline at end of file
+The TNT gets powered. In order to do this, the update that powers it needs to come from
```
Comment (EmiliaThorsen) [inline BugsAndExploits/TntDuping.md:19]: the powering needs to explicitly not update anything

File: BugsAndExploits/TntDuping.md [hunk 4]
```diff
@@ -6,15 +7,21 @@ description: Explaination of tnt duplication
...
+- a moving rail (1.12 and below) being replaced
+- anything that pop off from a state update (supporting block removed): coral fan is mostly used because it still create a moving block and can be reused.
+If it's a brittle block, that block must not be in front of a moved block because it will be placed in the broken block list that will be handled separately
+- a moving lit observer being replaced (works in all versions that have observers)
+
+These specific blocks are able to create a normal update after the list of blocks to be moved is created, but before some of the moved blocks have been removed.
```
Comment (EmiliaThorsen) [inline BugsAndExploits/TntDuping.md:25]: they work cus they give an update when moved or when something it is next too moves

**Review by EmiliaThorsen — COMMENTED** [review]
looks good, just had a bit ify stuff about tnt stuffs

File: BugsAndExploits/TntDuping.md (hunk 4 above)
Comment (pwouik) [reply BugsAndExploits/TntDuping.md:25]: it's the old blocks that cause updates, I precise here if it's a brittle block poping off (caused by a nearby block replacing the support block), or special updates when the state is replaced, I should expand the observer case to most components that can hard power (except detector rail)

Comment (fxmorin) [reply BugsAndExploits/TntDuping.md:25]: You could just write the order in which each action is done for pistons. Then explain why duping happens at the specific point

Comment (pwouik) [reply BugsAndExploits/TntDuping.md:25]: the actions are already detailed in the piston page,I should also put a link
redstone torch is included in components that hard power

**Comment by Earthcomputer** [comment]
Hi, pages have now been split to a separate repo, please transfer this pull request to https://github.com/TechMCDocs/pages ;)

**Pull request closed without merging**